src/core/Akka.MultiNodeTestRunner.Shared.Tests/MultiNodeTestRunnerDiscovery/DiscoverySpec.cs

```diff
@@ -46,7 +46,7 @@ public void Public_props_and_fields_are_considered()
 
         private static Dictionary<string, List<NodeTest>> DiscoverSpecs()
         {
-            using (var controller = new XunitFrontController(AppDomainSupport.IfAvailable, new System.Uri(Assembly.GetExecutingAssembly().CodeBase).LocalPath))
+            using (var controller = new XunitFrontController(AppDomainSupport.IfAvailable, new System.Uri(typeof(DiscoveryCases).GetTypeInfo().Assembly.CodeBase).LocalPath))
```

Comment:
If this passes in this PR, I'll revert my changes on https://github.com/akkadotnet/akka.net/pull/2875